lib/api/core/notifier.js

```diff
@@ -340,11 +340,16 @@ class NotifierController {
           result._source,
           result._id);
 
+        
+        const updatedFields = Object.keys(request.input.body)
+          .filter(_updatedFields => !['_kuzzle_info'].includes(_updatedFields));
```

Comment:
_kuzzle_info cannot be updated by users

```suggestion
          .filter(_updatedFields => _updatedFields !== '_kuzzle_info');
```